source/uwp/Renderer/idl/AdaptiveCards.Uwp.idl

```diff
@@ -1751,6 +1744,12 @@ namespace AdaptiveCards
 
             [propget] HRESULT ActionRenderers([out, retval] AdaptiveActionRendererRegistration** value);
 
+            [propget] HRESULT UserInputs([out, retval] AdaptiveInputs** value);
+
+            HRESULT AddInputItem([in] IAdaptiveCardElement* cardElement, [in] Windows.UI.Xaml.UIElement* uiElement);
+
+            HRESULT SendActionEvent([in] AdaptiveActionEventArgs* eventArgs);
```

Comment:
>SendActionEvent [](start = 20, length = 15)

We spoke about this offline.  This feels weird to me from an API design perspective because you're not going to actually call this method at render time.  Saving it for the click handler requires keeping the entire render context around for the click handler, at which point the other stuff in there doesn't make as much sense anymore.  Consider moving this method into a separate interface that you can get from the render context.